plutus-numeric/src/PlutusTx/NatRatio/Internal.hs

```diff
@@ -257,18 +259,18 @@ toRational (NatRatio r) = r
 
 {- | Newtype for deriving Schema and JSON instances
```

Comment:
An example of use would be good, especially since we've changed the type a bit. You can look at the examples in the other `plutus-numeric` Haddocks for inspiration.